Sources/Kevin/Accounts/BankSelection/KevinBankSelectionState.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 internal struct KevinBankSelectionState : IKevinState {
     let selectedCountry: String
     let selectedBankId: String?
+    let selectedCountryUnsupported: Bool
```

Comment:
why not checking just by bankItems? If bankItems is empty - show empty/error state. So I think we can omit this variable